config/module.config.php

```diff
@@ -31,6 +31,11 @@
             'AssetManager\Resolver\AliasPathStackResolver'      => 1000,
             'AssetManager\Resolver\PathStackResolver'           => 500,
         ),
+        'view_helper' => array(
+            'cache'            => null,
+            'query_string'     => '_',
+            'development_mode' => false,
```

Comment:
Nothing in the ZF2 default configs, as far as I know... do you prefer to rename it?